egs/wsj/s5/steps/nnet3/train_raw_dnn.py

```diff
@@ -286,7 +288,9 @@ def train(args, run_opts):
     # use during decoding
     common_train_lib.copy_egs_properties_to_exp_dir(egs_dir, args.dir)
 
-    if (args.stage <= -3) and os.path.exists(args.dir+"/configs/init.config"):
+    add_lda = common_train_lib.is_lda_added(config_dir)
+
```

Comment:
Are there other reasons we use init.config other than to make the LDA-like transform?